sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +112,49 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  private def getRevisionFromURL(jarURL: URL): String = {
+    scala.io.Source.fromURL(jarURL).getLines().toSeq
+      .filter(_.startsWith("revision="))
+      .map(_.split("=").last)
+      .headOption
+      .getOrElse("UNKNOWN")    
+  }
+
+  private def detectMultipleJar(propFileName: String, jarName: String, conf: RapidsConf): Unit = {
+    val classloader = ShimLoader.getShimClassLoader()
+    val possibleRapidsJarURLs = classloader.getResources(propFileName).asScala.toSeq
+
+    val revisionMap: Map[String, Seq[URL]] = possibleRapidsJarURLs.map { url =>
+      val revision = getRevisionFromURL(url)
+      (revision, url)
+    }.groupBy(_._1).mapValues(_.map(_._2)).toMap
+    lazy val rapidsJarsVersMsg = revisionMap.map {
+      case (revision, urls) => {
+        s"revison: $revision\n" + urls.map {
+          url => "jar URL: " + url.toString.split("!").head + "\n" + 
```

Comment:
nit:
Let's indent the jar URL lines. Then we can quickly get how many versions are detected.
```suggestion
          url => "    jar URL: " + url.toString.split("!").head + "\n" + 
```

Reply:
done.